app/carpool/views.py

```diff
@@ -149,7 +149,8 @@ def start_geojson():
                 'leave_time_human': pool.leave_time.strftime(dt_format),
                 'return_time_human': pool.return_time.strftime(dt_format),
                 'driver_gender': escape(pool.driver.gender),
-                'is_approximate_location': is_approximate_location
+                'is_approximate_location': is_approximate_location,
+                'hidden': escape(pool.destination.hidden)
```

Comment:
You shouldn't need an `escape()` here because `hidden` is a boolean and we don't have to worry about user contributed text sneaking in here.